README.md

```diff
@@ -1,9 +1,57 @@
-# TIC TAC TOE
+## TIC TAC TOE
 This is the third project of the Main Ruby curriculum at Microverse. The goal is to Build a tic-tac-toe game on the command line where two human players can play against each other and the board is displayed in between turns.
+* [Authors](#authors)
+* [Technologies](#technologies)
+* [Setup](#setup)
+* [Instructions](#instructions)
+* [Winnning Patterns](#winning-patterns)
 
-# Authors
-## Raymond Oluoch https://github.com/rOluochKe/
-## Andres F. Moya https://github.com/AndresFMoya/
+## Authors
+This project was contibuted by
+* Raymond Oluoch https://github.com/rOluochKe/
+* Andres F. Moya https://github.com/AndresFMoya/
+	
+## Technologies
+Project is created with:
+* Ruby version: 2.5.5
+	
+## Setup
+To run this project, install it locally using npm:
 
-# 1. Usage Instructions
-Clone the repo and run the app: ruby 'bin/main.rb'. The game accepts only integers from 1 to 9. The player who succeeds in placing three of their marks in a row wins the game, or it's draw.
\ No newline at end of file
+```
+$ clone the repo: git clone https://github.com/rOluochKe/tictactoe.git
+$ change directory: cd tictactoe
+$ run the program: ruby 'bin/main.rb'
+```
```

Comment:
separating the code into chunks:

```
$ clone the repo: git clone https://github.com/rOluochKe/tictactoe.git
```
```
$ change directory: cd tictactoe
```

```
$ run the program: ruby 'bin/main.rb'
```

and adding some comments in between lines, would give a better presentation and make it easier for a user with little programming knowledge.